### PR TITLE
New package: MigrosBank.EBanking version 9.2.2812

### PR DESCRIPTION
**Pull request by noraa-junker**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122915)

### CODE REVIEW

**Comment by Aaron-Junker** [comment]
I don't get why it's failing

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 
2023-10-17 03:37:33.597 [CLI ] ShellExecute installer failed: 1 

image

**Comment by stephengillie** [comment]
Installer switches:
```
Usage: C:\Users\User\AppData\Local\Temp\WinGet\MigrosBank.EBanking.9.2.2812\appwinpc.exe [optiPS C:\Users\User\Desktop>
Options:
  -?, -h, --help                           Displays this help.
  --version                                Displays version information.
  --keepCredentials                        Store the credentials information.
                                           Dont delete them
  --uninstalllogs                          creates the UninstallLogs file.
  --deleteDirs <[]>                        Delete the directory
  --framework-version                      Displays the version of the Qt
                                           Installer Framework.
  -v, --verbose                            Verbose mode. Prints out more
                                           information.
  --proxy                                  Use system proxy on Windows and
                                           Linux. This option has no effect on
                                           macOS.
  --no-proxy                               Do not use system proxy.
  --script <file>                          Execute the script given as
                                           argument.
  --checkupdates                           Check for updates and return an XML
                                           description.
  --updater                                Start application in updater mode.
  --manage-packages                        Start application in package manager
                                           mode.
  --no-force-installations                 Allow deselecting components that
                                           are marked as forced.
  --no-size-checking                       Disable checking of free space for
                                           installation target.
  --show-virtual-components                Show virtual components in installer
                                           and package manager.
  --logging-rules <rules>                  Enables logging according to passed
                                           rules. Comma separated logging rules
                                           have the following syntax:
                                           loggingCategory=true/false. Passing
                                           empty logging rules enables all
                                           logging categories. The following
                                           rules enable a single category:
                                           ifw.*=false,ifw.category=true The
                                           following logging categories are
                                           available:
                                           ifw.componentChecker
                                           ifw.resources
                                           ifw.translations
                                           ifw.network
  --create-local-repository                Create a local repository inside the
                                           installation directory. This option
                                           has no effect on online installers.
  --addRepository <URI,...>                Add a local or remote repository to
                                           the list of user defined
                                           repositories.
  --addTempRepository <URI,...>            Add a local or remote repository to
                                           the list of temporary available
                                           repositories.
  --setTempRepository <URI,...>            Set a local or remote repository as
                                           temporary repository, it is the only
                                           one used during fetch.
                                           Note: URI must be prefixed with the
                                           protocol, i.e. file:///, https://,
                                           http:// or ftp://.
  --startserver <mode,socketname,key>      Starts the application as headless
                                           process waiting for commands to
                                           execute. Mode can be DEBUG or
                                           PRODUCTION. In DEBUG mode, the option
                                           values can be omitted.Note: The
                                           server will not shutdown on his own,
                                           you need to quit the process by hand.
  --startclient <socketname,key>           Starts the application to debug the
                                           client-server communication. If a
                                           value is omitted, the client will use
                                           a default instead. Note: The server
                                           process is not started by the client
                                           application in that case, you need to
                                           start it on your own.
  --installCompressedRepository <URI,...>  Installs QBSP or 7z file. The QBSP
                                           (Board Support Package) file must be
                                           a .7z file which contains a valid
                                           repository.
  --silentUpdate                           Updates all packages silently.
  --platform <plugin>                      Use the specified platform plugin.
  --squish-port <port number>              Give a port where Squish can connect
                                           to. If no port is given, default port
                                           11233 is used. Note: To enable Squish
                                           support you first need to build IFW
                                           with SQUISH_PATH parameter where
                                           SQUISH_PATH is pointing to your
                                           Squish installation folder:
                                           <path_to_qt>/bin/qmake -r
                                           SQUISH_PATH=<pat_to_squish>

Arguments:
  Key=Value                                Key Value pair to be set.


```

File: manifests/m/MigrosBank/EBanking/9.2.2812/MigrosBank.EBanking.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: MigrosBank.EBanking
+PackageVersion: 9.2.2812
+InstallerType: exe
+InstallerSwitches:
+  Silent: --silentUpdate
```
Comment (stephengillie) [inline manifests/m/MigrosBank/EBanking/9.2.2812/MigrosBank.EBanking.installer.yaml:8]:
```suggestion
  Silent: --silent
```
Had an urge to see if this undocumented switch would work, and yes, it results in the unattended install that we're seeking.

**Comment by stephengillie** [comment]
The [FAQ](https://www.migrosbank.ch/de/hilfe/neues-ebanking/faq-fragen-ebanking.html?t=sicherheit) satisfies some of the questions in the [Policy 1.8](https://learn.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies#18-financial-transactions) check.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Caught exception while trying to kill child processes: System.AggregateException: Not all processes in process tree could be terminated. (Access is denied.)  ---

**Review by stephengillie — APPROVED** [review]

**Comment by mdanish-kh** [comment]
@Aaron-Junker You'll have to mark the above conversation as resolved for the auto-merge trigger to merge this PR.

**Pull request merged**